ext/hiera/hiera.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: 5
+defaults:
+  # The default value for "datadir" is "data" under the same directory as the hiera.yaml
+  # file (this file)
+  # When specifying a datadir, make sure the directory exists.
+  # See https://docs.puppet.com/puppet/4.10/environments.html for further details on environments.
+  datadir:
```

Comment:
This is wrong. Did you perhaps intend it to be like this?
```
# datadir: data
```
Now it is a yaml syntax error (key without a value)